internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -43,10 +44,10 @@ macro(build_knowhere)
     endif ()
 
     externalproject_add(knowhere_ep
-            # GIT_REPOSITORY  "https://github.com/milvus-io/knowhere.git"
-            # GIT_TAG         main
-            URL             ${KNOWHERE_SOURCE_URL}
-            URL_MD5         ${KNOWHERE_SOURCE_MD5}
+            GIT_REPOSITORY  "https://github.com/milvus-io/knowhere.git"
+            GIT_TAG         main
+            # URL             ${KNOWHERE_SOURCE_URL}
+            # URL_MD5         ${KNOWHERE_SOURCE_MD5}
```

Comment:
For stability, please release a new version of knowhere and set the URL & MD5 according to the version.